RELEASE.md

```diff
@@ -0,0 +1,19 @@
+# Release
+
+To release new version of Kuma follow the steps:
+
+1. Create a CHANGELOG.md.
+2. Create PR to [kuma.io website repository](https://github.com/Kong/kuma-website) making sure that all new features
```

Comment:
I don't think that documenting all new features is part of the release process

Reply:
It should be done when developing new features, but release it's a good time (since we modify `CHANGELOG.md`) to double-check that we've done it